spec/lib/hubspot/company_properties_spec.rb

```diff
@@ -82,15 +82,13 @@
 
     describe '.create!' do
       context 'with no valid parameters' do
-        cassette 'fail_to_create_property'
```

Comment:
Should this be replaced with something?

Reply:
Great question. This cassette does not exist. I'm surprised VCR doesn't complain since this cassette doesn't exist. In this test, we don't need to record the request because the `create!` functionality, when given an empty hash, returns early with `nil` and never makes an API call.